api/src/models/pitch.ts

```diff
@@ -0,0 +1,105 @@
+import mongoose, { Document, Schema } from 'mongoose';
+import { IUser } from './user';
+
+export interface IPitch extends Document<any> {
+  name: string;
+  pitchAuthor: mongoose.Types.ObjectId;
+  pitchStatus: string;
+  assignmentStatus: string;
+  assignmentGoogleDocLink: string;
+  assignmentContributors: [IUser];
+  topic: string;
+  currentWriters: number;
+  targetWriters: number;
+  currentEditors: number;
+  targetEditors: number;
+  currentData: number;
+  targetData: number;
+  currentVisuals: number;
+  targetVisuals: number;
+  currentIllustration: number;
+  targetIllustration: number;
+  currentPhotography: number;
+  targetPhotography: number;
+  currentFactChecking: number;
+  targetFactChecking: number;
+  currentRadio: number;
+  targetRadio: number;
+  currentLayout: number;
+  targetLayout: number;
+  approvedBy: IUser;
+  similarStories: [string];
+  deadline: Date;
+}
+
+const pitchStatusEnum = {
+  APPROVED: 'APPROVED',
+  PENDING: 'PENDING',
+  REJECTED: 'REJECTED',
+  NONE: 'NONE',
+};
+
+const assignmentStatusEnum = {
+  IN_PROGRESS: 'IN_PROGRESS',
+  COMPLETED: 'COMPLETED',
+  ABANDONED: 'ABANDONED',
+  NONE: 'NONE',
+};
+
+// TODO: don't repeat this code
+const interestsEnum = {
+  POLITICS: 'POLITICS',
+  EDUCATION: 'EDUCATION',
+  HOUSING: 'HOUSING',
+  LIT: 'LIT',
+  MUSIC: 'MUSIC',
+  VISUAL_ARTS: 'VISUAL ARTS',
+  STAGE_AND_SCREEN: 'STAGE AND SCREEN',
+  FOOD_AND_LAND: 'FOOD AND LAND',
+  NATURE: 'NATURE',
+  TRANSPORTATION: 'TRANSPORTATION',
+  HEALTH: 'HEALTH',
+  CANNABIS: 'CANNABIS',
+  IMMIGRATION: 'IMMIGRATION',
+  FUN: 'FUN',
+  NONE: 'NONE',
+};
+
+/**
+ * Mongoose Schema to represent a Pitch at South Side Weekly
+ */
+const Pitch = new mongoose.Schema({
+  name: { type: String, default: null, required: true },
+  pitchAuthor: { type: Schema.Types.ObjectId, ref: 'User', required: true },
+  pitchStatus: { type: pitchStatusEnum, default: pitchStatusEnum.NONE },
+  assignmentStatus: {
+    type: assignmentStatusEnum,
+    default: assignmentStatusEnum.NONE,
+  },
+  assignmentGoogleDocLink: { type: String, default: null },
+  assignmentContributors: [{ type: Schema.Types.ObjectId, ref: 'User' }],
+  topic: { type: interestsEnum, default: interestsEnum.NONE },
+  currentWriters: { type: Number, default: 0 },
+  targetWriters: { type: Number, default: 100 },
```

Comment:
Actually now that I think about it, it might make more sense to set the default target numbers to 0 - that way if the user doesn't input anything on the frontend it will default to 0 which is more intuitive